config/sunspot.yml

```diff
@@ -0,0 +1,25 @@
+production:
+  solr:
+    hostname: localhost
+    port: 8983
+    log_level: WARNING
+    path: /solr/production
+    # read_timeout: 2
+    # open_timeout: 0.5
+
+development:
+  solr:
+    hostname: localhost
+    port: 8982
+    log_level: INFO
+    path: /solr/development
+
+test: &TEST
+  solr:
+    hostname: localhost
+    port: 8981
+    log_level: WARNING
+    path: /solr/test
+
+cucumber:
```

Comment:
Good catch; this was part of a solution I had tried for getting stubbing to work in cucumber, but turned out not to be needed. I'll go ahead and revert it.